src/phy/fuzz_injector.rs

```diff
@@ -0,0 +1,120 @@
+use Result;
+use phy::{self, DeviceCapabilities, Device};
+use time::Instant;
+
+// This could be fixed once associated consts are stable.
+const MTU: usize = 1536;
+
+/// Represents a fuzzer. It is expected to either generate or contain data, which it replaces bytes
```

Comment:
Contain data?

Reply:
That was a comment from a previous version. I will remove it.